tests/bsim/bluetooth/host/misc/sample_test/test_scripts/run.sh

```diff
@@ -62,8 +62,10 @@ cd ${BSIM_OUT_PATH}/bin
 
 # Instantiate all devices in the simulation.
 # The `testid` parameter is used to run the right role or procedure (here "dut" vs "tester").
-Execute "${test_exe}" -v=${verbosity_level} -s=${simulation_id} -d=0 -rs=420 -testid=dut
-Execute "${test_exe}" -v=${verbosity_level} -s=${simulation_id} -d=1 -rs=69  -testid=peer
+Execute "${test_exe}" -v=${verbosity_level} -s=${simulation_id} -d=0 -rs=420 -testid=dut \
+		-argstest log_level 3
+Execute "${test_exe}" -v=${verbosity_level} -s=${simulation_id} -d=1 -rs=69  -testid=peer \
```

Comment:
We could set this from a shell variable declared at the top. E.g. `-argstest log_level ${test_log_level}`. Can be done in a future PR :+1: